src/main/java/hw6/pages/MetalAndColorsPage.java

```diff
@@ -0,0 +1,214 @@
+package hw6.pages;
+
+import com.epam.jdi.light.elements.complex.Checklist;
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JDropdown;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.XPath;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.Checkbox;
+import com.epam.jdi.light.ui.html.elements.common.Text;
+import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+
+public class MetalAndColorsPage extends WebPage {
+
+    // Colors dropdown
```

Comment:
You shold not live a comment like this. What is the reason of it ? 
All of us can realise that **colorsDropdown** field is **Colors dropdown**

Reply:
Removed comments like that